### PR TITLE
Week 49 Updates

### PR DESCRIPTION
**Pull request by Br4nd0R**
* This is to fix the currently open security alert for `glob-parent`
* Not currently updating python based dependencies do to a version limitation in CumulusCI.

### CODE REVIEW

**Comment by Br4nd0R** [comment]
Waiting for a new release of snowfakery before merging this.

**Review by jstvz — APPROVED** [review]

**Pull request merged**
